doc/requirements.txt

```
@@ -3,25 +3,30 @@ pip
 appdirs
 autograd
 autoray
-jax==0.2.19
-jaxlib==0.1.71
+jax==0.3.14
+jaxlib==0.3.14
 mistune==0.8.4
 m2r2
 numpy
 pygments-github-lexers
 semantic_version==2.10
 scipy
 docutils==0.16
-sphinx==3.5.3
+sphinx==3.5.3; python_version < "3.10"
+sphinx==5.0.2; python_version == "3.10"
 sphinx-automodapi==0.13
 sphinx-copybutton
-sphinxcontrib-bibtex==0.4.2
-tensorflow==2.6.5
+sphinxcontrib-bibtex==0.4.2; python_version < "3.10"
+sphinxcontrib-bibtex==2.4.2; python_version == "3.10"
+tensorflow==2.6.5; sys_platform != "darwin"
```

Got `ERROR: No matching distribution found for tensorflow==2.6.5` on Python 3.10.
```suggestion
tensorflow==2.6.5; sys_platform != "darwin" and python_version < "3.10"
tensorflow==2.8.0; sys_platform != "darwin" and python_version == "3.10"
```